src/app/CoordinateSectorCreator.js

```diff
@@ -0,0 +1,382 @@
+        //      coorindate form elements
+        _panelController: null,
+        // zoomLevel: Number
+        // summary:
+        //      the cache level to zoom the map to
+        zoomLevel: 13,
+
+        // Properties to be sent into constructor
+        // map: esri/map
+        // summary:
+        //      the map to zoom
+        map: null,
+        symbol: null,
+        graphicsLayer: null,
+
+        postCreate: function () {
+            // summary:
+            //      Overrides method of same name in dijit._Widget.
+            console.log('app.CoordinateSectorCreator::postCreate', arguments);
+            if (!this.map) {
+                throw 'This widget requires an esri/map to be useful.';
+            }
+            this.symbol = new SimpleFillSymbol().setColor(null).outline.setColor('red');
+            this.graphicsLayer = new GraphicsLayer({ id: 'sectors' });
+            this.map.addLayer(this.graphicsLayer);
+
+            this._panelController = {
+                panels: {
+                    // utm: this.utmNode,
+                    // dm: this.dmNode,
+                    // dms: this.dmsNode,
+                    dd: this.ddNode
+                },
+                hideAllBut: function (showMe) {
+                    for (var prop in this.panels) {
+                        if (this.panels.hasOwnProperty(prop)) {
+                            if (showMe && prop === showMe) {
+                                domClass.replace(this.panels[prop], 'show', 'hide');
+                                this.visible = this.panels[prop];
+                                continue;
+                            }
+
+                            domClass.replace(this.panels[prop], 'hide', 'show');
+                        }
+                    }
+                },
+                visible: this.ddNode
+            };
+
+            this.set('valid', false);
+
+            this.setupConnections();
+
+            this.inherited(arguments);
+        },
+        zoom: function () {
+            // summary:
+            //      zooms the map to the point created by _getCoverageParams
+            //  summary:
+            //      the point created by the user input or returned by
+            //      the geometry service
+            console.log('agrc.widgets.locate.ZoomToCoords::zoom', arguments);
+            console.log(this.map.spatialReference.wkid);
+
+            if (!this.map) {
+                throw 'This widget requires an esri/map to be useful.';
+            }
+
+            // disable zoom button
+            domClass.add(this.zoomNode, 'disabled');
+            domAttr.set(this.zoomNode, 'disabled', true);
+
+            // reset errors
+            //domClass.remove(this.errorNode, ['alert', 'alert-danger', 'text-center']);
+            //this.errorNode.innerHTML = '';
+            var coverageParams = this._getCoverageParams();
+            var point = coverageParams.center;
+
```

Comment:
Should you clear any previous graphics before creating the new one? Or if the client wants to be able to see more than one at the same time should you have a clear graphics button?